File: Redis_PoC/python/code/sample3.py
import redis
import time
import os

redis_ip = os.environ.get("REDIS_SERVER_ADDRESS","127.0.0.1")
redis_port = os.environ.get("REDIS_SERVER_PORT",6379)

r = redis.StrictRedis(host= redis_ip, port=redis_port)

# simple set,increase
pipe = r.pipeline()
my_list = { "key1":"value1", "foo":"bar", "Num":123 }
r.mset(my_list)


print r.mget('key1','foo',"Num")

File: Redis_PoC/php/code/sample3.php
<?php
require "vendor/autoload.php";
Predis\Autoloader::register();

$redis_ip = "127.0.0.1";
if (getenv("REDIS_SERVER_ADDRESS"))
  $redis_ip = getenv("REDIS_SERVER_ADDRESS");

$redis_port = 6379;
if (getenv("REDIS_SERVER_PORT"))
  $redis_port = getenv("REDIS_SERVER_PORT");

$redis = new Predis\Client(array('host' => $redis_ip, 'port' => $redis_port));


$my_list = array("key1"=>"value1","foo"=>"bar","Num"=>123);
$redis->mset($my_list);

print_r($redis->mget('key1','foo','Num'));
?>
